envisage/service.py

```diff
@@ -15,7 +15,7 @@
 
 # Enthought library imports.
 from traits.api import TraitType
-
+from traits.trait_errors import TraitError
```

Comment:
Let's import from `traits.api`. (Most of the ETS packages are designed so that you should only need to import from the `api` modules; that way, the internal details can be changed without risking breaking client code.)